pkg/controller/servicebindingrequest/reconciler.go

```diff
@@ -19,9 +22,11 @@ import (
 
 // Reconciler reconciles a ServiceBindingRequest object
 type Reconciler struct {
-	client    client.Client     // kubernetes api client
-	dynClient dynamic.Interface // kubernetes dynamic api client
-	scheme    *runtime.Scheme   // api scheme
+	client            client.Client     // kubernetes api client
+	dynClient         dynamic.Interface // kubernetes dynamic api client
+	scheme            *runtime.Scheme   // api scheme
+	ensureWatchingGVR func(gvr schema.GroupVersionResource) error
+	restMapper        meta.RESTMapper
```

Comment:
Why is this `restMapper` added here? I don't see it anywhere being used for `Reconcile`.

Reply:
You are right. We should add `restMapper` to `SBRController` only.